frontApp/app/profile.tsx

```diff
@@ -0,0 +1,221 @@
+                    <View style={styles.profileView}>
+                        <View style={styles.iconSettings}>
+                            <MaterialIcons name="phone" size={32} color="black" />
+                        </View>
+                        <TextInput style={styles.accountText} value = "Phone" editable = {false}/>
+                    </View>
+                    <View style={styles.profileView}>
+                        <View style={styles.iconSettings}>
+                            <MaterialCommunityIcons name="onepassword" size={32} color="black" />
+                        </View>
+                        <TextInput style={styles.accountText} value = "Password" editable = {false}/> 
+                    </View>
+                </View>
+
+                <View>
+                    <Text style={styles.profileVerTitleText}>Verification</Text>
+                        <View style={styles.profileView}>
+                            <View style={styles.iconSettings}>
+                                <Ionicons name="checkmark-circle-sharp" size={32} color="black" />
+                            </View>
+                            <TextInput style={styles.linkText} value = "Verify my Email" editable = {false}/> 
+                        </View>
+                        <View style={styles.profileView}>
+                            <View style={styles.iconSettings}>
+                                <Ionicons name="checkmark-done-circle" size={32} color="black" />
+                            </View>
+                            <TextInput style={styles.linkText} value = "Verify my ID" editable = {false}/> 
+                        </View>
+                        <View style={styles.profileView}>
+                            <View style={styles.iconSettings}>
+                                <Ionicons name="checkmark-circle-sharp" size={32} color="black" />
+                            </View>
+                            <TextInput style={styles.linkText} value = "Verify my Phone" editable = {false}/> 
+                        </View>
+                </View>
+                <View style={styles.edtBtnTitleText} >
+                    <TouchableOpacity style={styles.edtBtn}>
+                        <View style={styles.profileView}>
+                            <Text style={styles.edtBtnText}>Edit Profile</Text>
+                            <Ionicons name="create" size={32} color="#808080" />
+                        </View>
+                    </TouchableOpacity>
+                </View>
+            </ScrollView>
+        </View>
+    );
+};
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: "#e0a5a5",
+        marginBottom: '0%',
+    },
+    header: {
+        flexDirection:"row",
+        marginBottom: '1%',
+        marginTop: '15%',
+        marginLeft: '33%'
+    },
+
+    header2: {
+        flex: 3
+    },
+    headerText: {
+        color: '#000000',
```

Comment:
We have been using White text for headings and titles. Let's stick to the color scheme of using the white text. The settings page has white text and the profile page has black text.

Reply:
updated text to black. spoke in person